backend/routers/plugins.py

```
@@ -109,3 +116,129 @@ def get_plugin_money_made(plugin_id: str):
+@router.post('/v2/plugins/disable')
+def disable_plugin_endpoint(plugin_id: str, uid: str = Depends(auth.get_current_user_uid)):
+    plugin = get_plugin_by_id_db(plugin_id, uid)
+    if not plugin:
+        raise HTTPException(status_code=404, detail='Plugin not found')
+    disable_plugin(uid, plugin_id)
+    decrease_plugin_installs_count(plugin_id)
+    return {'status': 'ok'}
+
+
+@router.get('/v3/plugins', tags=['v3'], response_model=List[Plugin])
+def get_plugins(uid: str = Depends(auth.get_current_user_uid), include_reviews: bool = True):
```

Ruff

Redefinition of unused `get_plugins` from line 60

(F811)